v1/message.go

```diff
@@ -139,15 +143,50 @@ func CreatePrivateTextMessage(data []byte, lastClock int64, chatID string) Messa
 	return createTextMessage(data, lastClock, chatID, MessageTypePrivate)
 }
 
-// DecodeMessage decodes a raw payload to Message struct.
-func DecodeMessage(data []byte) (message StatusMessage, err error) {
+func unwrapMessage(data []byte) (*StatusProtocolMessage, error) {
+	message := StatusProtocolMessage{}
```

Comment:
Nitpick: using `var message StatusProtocolMessage` you won't allocate memory if `data` is invalid.